doc/conf.py

```diff
@@ -168,6 +168,7 @@ def _check_dependencies():
     'thumbnail_size': (320, 224),
     'compress_images': ('thumbnails', 'images'),
     'matplotlib_animations': True,
+    'image_srcset': ["2x"],
```

Comment:
Maybe
```suggestion
    # 3.7 CI doc build should not use hidpi images in the testing phase
    'image_srcset': [] if platform.python_version[3:] == '3.7' else ["2x"],
```
That way we have builds with and without hidpi images (ok still using the same code but heh).

Reply:
Oh, I see.  Sure we can try that.

Reply:
You have `sys` already though: `sys.version_info[:2] == (3, 7)`